src/pipeline_integration.jl

```diff
@@ -0,0 +1,258 @@
+import Plots
+using Plots.RecipePipeline
+
+# Define overrides for RecipesPipeline hooks.
```

Comment:
Would it have been better to name the package `RecipesPipeline` (note the s)?

Reply:
I think so yes :+1:

Reply:
Sorry, that comment was from a while ago, and I always slip and call it RecipesPipeline because of RecipesBase.  If we want to rename, it's best to do it now...

Reply:
Agreed - we had to rename StatPlots to StatsPlots at some point, and it was a bit of a nightmare